README.md

```diff
@@ -17,16 +17,27 @@ There is also a requirements file which pretty much does the same via `pip insta
 From the command line:
 
 ```
-git clone https://github.com/satyarth/pixelsort.git
-cd pixelsort
-python3 pixelsort.py %PathToImage% [options]
+pip install pixelsort
+python3 -m pixelsort %PathToImage% [options]
 ```
 
 Tip: To replicate Kim Asendorf's original [processing script](https://github.com/kimasendorf/ASDFPixelSort), first sort vertically and then horizontally in `threshold` (default) mode:
 
 ```
-python3 pixelsort.py %PathToImage% -a 90
-python3 pixelsort.py %PathToSortedImage%
+python3 -m pixelsort %PathToImage% -a 90
+python3 -m pixelsort %PathToSortedImage%
+```
+
+As a package:
+
+```
+>>> from pixelsort.pixelsort import pixelsort
```

Comment:
@ethanhs is there any way to make importing this main function from this package easier? Currently this is the way to write it out since you need to access the pixelsort function in the pixelsort file in the pixelsort module.

Reply:
I think the best thing to do in this case would be to rename the `pixelsort` module (perhaps `main.py` or `api.py`) then re-exporting it via the `__init__.py` (e.g. in the `__init__.py` put `from pixelsort.api import pixelsort`). Alternatively you could move/consolidate the function into another module if that is more appropriate (I'm not really familiar with the code).